nautobot/ipam/tests/test_api.py

```diff
@@ -443,6 +443,25 @@ def test_list_available_prefixes(self):
         for i, p in enumerate(response.data):
             self.assertEqual(p["prefix"], str(available_prefixes[i]))
 
+    def test_prefix_display_value(self):
+        """
+        Test that the `display` field is correctly populated.
+        """
+        prefix = (
+            Prefix.objects.filter(ip_version=6)
+            .filter(prefix_length__lt=128)
+            .exclude(type=choices.PrefixTypeChoices.TYPE_CONTAINER)
+            .first()
+        )
+        if prefix is None:
+            self.fail("Suitable prefix fixture not found")
```

Comment:
What's the purpose of this? You're listing all prefixes below, what is this specific `prefix` used for?

Reply:
Copy-pasta error. I will remove it. 🫢